.github/workflows/service_test_dropbox.yml

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Service Test Dropbox
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "core/src/**"
+      - "core/tests/**"
+      - "!core/src/docs/**"
+      - "!core/src/services/**"
+      - "core/src/services/dropbox/**"
+      - ".github/workflows/service_test_dropbox.yml"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  dropbox:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' || !github.event.pull_request.head.repo.fork
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup
+
+      - name: Load secret
+        id: op-load-secret
+        uses: 1password/load-secrets-action@v1
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          OPENDAL_DROPBOX_TEST: op://services/dropbox/test
+          OPENDAL_DROPBOX_ROOT: op://services/dropbox/root
+          OPENDAL_DROPBOX_REFRESH_TOKEN: op://services/dropbox/refresh_token
+          OPENDAL_DROPBOX_CLIENT_ID: op://services/dropbox/client_id
+          OPENDAL_DROPBOX_CLIENT_SECRET: op://services/dropbox/client_secret
+
+      - name: Test
+        shell: bash
+        working-directory: core
+        # It's easily for dropbox to trigger too_many_write_operations error.
```

Comment:
We need to retry this error.

Reply:
Is it easy to retry it in CI workflow?
What we should do is just change whether this error retries in parse_error?

Reply:
> Is it easy to retry it in CI workflow?

I mean it should be a temporary error.